table/block_based/block.cc

```diff
@@ -438,8 +429,8 @@ void DataBlockIter::SeekForPrev(const Slice& target) {
     return;
   }
   SeekToRestartPoint(index);
-  // Linear search (within restart block) for first key >= seek_key
 
+  // Linear search (within restart block) for first key >= seek_key
```

Comment:
Nothing really changed in this file? If it is not too hard, I suggest we separate code cleaning up from this unrelated feature.

Reply:
Done.